dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -111,7 +111,12 @@ public void store(ModelData modelData) {
             if (!id.isPresent()) {
                 LOGGER.warn("Cannot load entity from file since it's missing an id: " + modelData.toJson());
             } else {
-                if (fetch(kind, id.get()) == null) {
+                WithId prev = null;
+                try {
+                    prev = fetch(kind, id.get());
+                } catch (RuntimeException e) {
```

Comment:
I was thinking that we could get a error if the rest object can't be fetched due to a schema change.  But honestly, I think we would need to do a delete in catch to try to reset it properly.